incskill_site/templates/base.html

```diff
@@ -1,56 +1,37 @@
 {% load static %}
+
 <!DOCTYPE html>
 
 <html lang="en">
 
-<head>
-  <meta charset='utf-8'>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{% block title %}Incskill{% endblock %}</title>
-  <link rel="stylesheet" href="{% static 'css/style.css' %}" />
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
-  <script src="{% static 'js/jquery-1.11.1.js' %}"></script>
-  <script src="{% static 'js/rango-jquery.js' %}"></script>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
-</head>
-
-
-
-
-
-<body>
-  <!--request.user.is_authenticated apparently not compatible, despite being reccomended-->
-  {% if user.is_authenticated %}
-  <nav aria-label="Navigation Bar" aria-hidden="true">
-    <div class="topnav">
-      <a class="navbar-brand" href="#"><img src="{% static 'media/incskill.png' %}" alt="incskill logo" width="100"
-          height="auto" style="margin-top: -9px;"></a>
-      <a class="navbar-brand" href="{% url 'courses' %}">My Courses</a>
-
-
-      <a class="navbar-brand" href="{% url 'profile' %}"> {{user.username}} </a>
-
-      <div class="collapse navbar-collapse" id="navbarNav">
+  <head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% block title %}Incskill{% endblock %}</title>
+    <link rel="stylesheet" href="{% static 'css/style.css' %}" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="{% static 'js/jquery-1.11.1.js' %}"></script>
+    <script src="{% static 'js/rango-jquery.js' %}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+  </head>
+
+
+  <body>
+    <!--request.user.is_authenticated apparently not compatible, despite being reccomended-->
+    <nav aria-label="Navigation Bar" aria-hidden="true">
+      <div class="topnav">
+        <a class="navbar-brand" href="#"><img src="{% static 'media/incskill.png' %}" alt="incskill logo" width="100" height="auto" style="margin-top: -10.32%;"></a>
```

Comment:
what is this magic number `style="margin-top: -10.32%;"` does it work on all screen sizes and is there a better way to position this